examples/plot_simulate_gamma.py

```diff
@@ -44,44 +59,35 @@
 dpls = simulate_dipole(net)
 
 ###############################################################################
-# The network requires some time to reach steady state. Hence, we omit the
-# first 50 ms in our time-frequency analysis.
+# Take a look at how different cell types respond to the exogenous drive. Note
+# the periodic firing pattern of all cell types. While the basket cells fire
+# relatively synchronously, the pyramidal cell populations display a more
+# varied pattern, in which only a fraction of cells reach firing threshold.
+net.cell_response.plot_spikes_raster()
+
+###############################################################################
+# To confirm that the periodicity observed in the firing patterns correspond to
+# a population oscillation in the gamma-range, we can plot the time-frequency
+# representation together with the signal. Note that the network requires some
+# time to reach steady state. Hence, we omit the first 50 ms in our analysis.
 
-tstart = 50
+tmin = 50
 trial_idx = 0  # pick first trial
-mask = dpls[trial_idx].times > tstart
-times = dpls[trial_idx].times[mask]
-data = dpls[trial_idx].data['agg'][mask]
+decim = 8  # decimate plots by a factor of 8 (from 40 to 5 kHz)
```

Comment:
do we really need to do this? I think the fewer the parameters ... the better

Reply:
My plotting is insanely slow without some serious decimating, can you test the interactive performance on yours?

Reply:
It's not "insanely slow" but yeah the `decim` does make it faster. I can live with it :)

```py
In [6]: %timeit plot_tfr_morlet(dpls[trial_idx], freqs=freqs, n_cycles=7, tmin=t
   ...: min, decim=8, ax=axes[1])                                               
/Users/mainak/Documents/github_repos/hnn-core/hnn_core/viz.py:378: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
  cbar_ax = fig.add_axes([l + w + 0.05, cb_b, 0.03, cb_h])
140 ms ± 5.53 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)

In [7]: %timeit plot_tfr_morlet(dpls[trial_idx], freqs=freqs, n_cycles=7, tmin=t
   ...: min, decim=1, ax=axes[1])                                               
/Users/mainak/Documents/github_repos/hnn-core/hnn_core/viz.py:378: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
  cbar_ax = fig.add_axes([l + w + 0.05, cb_b, 0.03, cb_h])
940 ms ± 14.3 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Reply:
Mine ran with a reasonable execution time regardless of decimation as well, ~39 ms with decimation vs. ~295 ms without. For simplicity's sake, I'm not convinced we need an extra line defining the `decim` variable (just like we don't define a separate var for `n_cycles`) but that's stylistic.